CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### 2.15.1 (2023-08-07)
+
```

Comment:
Should be something in here. At least a hint that we now serve a new scoped package.